pkg/platform/azure/azure.go

```diff
@@ -180,9 +186,63 @@ func (a *AzurePlatform) GatherCloudletInfo(ctx context.Context, info *edgeproto.
 					if err != nil {
 						return fmt.Errorf("failed to parse %s value %s for flavor %s, %s", *cap.Name, *cap.Value, *v.Name, err)
 					}
-					flavor.PropMap = map[string]string{
-						"gpu": fmt.Sprintf("gpu:%d", num),
+					// See https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/overview?tabs=breakdownseries%2Cgeneralsizelist%2Ccomputesizelist%2Cmemorysizelist%2Cstoragesizelist%2Cgpusizelist%2Cfpgasizelist%2Chpcsizelist#gpu-accelerated
```

Comment:
I guess this is current naming....and we'll need to update this periodically.

Reply:
Yup, if they add another set of GPU flavors we'll need to update it.